README.md

```diff
@@ -1,27 +1,232 @@
 
-[badges]
-[description]
-[table of contents]
-[content ideas: link to deployed app, setup, test suite, GQL queries and mutations, database schema and description of each table, contributors]
+This [GraphQL on Rails API](insert live endpoint link) serves queries and mutations to [Eras](insert live FE link), an application that allows you to visualize your life holistically in order to live more intentionally.
+
+By providing info on personal events, you will be shown a visual calendar of your life. By viewing the seemingly fleeting weeks that add up to years and decades, you are able to appreciate the little moments, cherish the time you have with loved ones, and live with more intention.
+
+With Eras, you'll see your life in a whole new way.
+
+[insert screenshots]
+
+## Readme Content
+- [User Interface](#user-interface)
+- [Local Setup](#local-setup)
+- [Test Suite](#test-suite)
+- [GraphQL Schema](#graphql-schema)
+- [Database Schema](#database-schema)
+- [Project Board](#project-board)
+- [Contributors](#contributors)
+
+## User Interface
+- [Deployed application](insert link)
+- [Frontend repo](https://github.com/Turing-Eras/front_end)
+  - Stack: React, Typescript
 
 ## Local Setup
 - Versions
   - Rails 6.1.0
   - Ruby 2.5.3
 - Fork and clone the repository
-- `cd` in your local repo version and run the following commands to install gems and setup database:
-  - `bundle` (if this fails, try to `bundle update` and then retry)
-  - `rails db:create`
-  - `rails db:migrate`
-  - `rails db:seed`
-  - Create a manifest.js file nested under app/assets/config and place the following inside:
-    ```
-    //= link graphiql/rails/application.css
-    //= link graphiql/rails/application.js
-    ```
-    - This file is ignored by git tracking but is necessary to run the server locally.
+- `cd` in your local repo version and run the following commands
+  - To install gems:
+    -  `bundle` (if this fails, try to `bundle update` and then retry)
+  - To setup database:
+    - `rails db:create`
+    - `rails db:migrate`
+    - `rails db:seed`
+- Create a manifest.js file nested under app/assets/config and place the following inside:
+  ```
+  //= link graphiql/rails/application.css
+  //= link graphiql/rails/application.js
+  ```
+  - This file is ignored by git tracking but is necessary to run the server locally.
 - Run your own development server:
   - In config/application.rb, uncomment out line 15 (`require "sprockets/railtie"`)
   - `rails s`
-  - You should be able to access the GraphQL interface via http://localhost:3000/graphiql
+  - You should be able to access the GraphQL interface and see available queries and mutations via the docs on http://localhost:3000/graphiql
+
+## Test Suite
+- Run with `bundle exec rspec`
+- All tests should be passing
+- 83.24% test coverage
+
+
+## GraphQL Schema
+- Queries available:
+  - Get Single Era: <details>
+    ```
```

Comment:
The Prettier way of doing this would be to do on line 57:
```
 - <details><summary>Get Single Era:</summary>
```
But for some reason the notes between the ` ``` ` aren't being formatted in the correct way.  This is the compromise.